packages/amazonq/.changes/next-release/Feature-0544c5fd-89fa-4290-a5f1-283372d6cf79.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "feat(chat): Add fsRead tool for Amazon Q Agentic Chat"
```

Comment:
you probably don't need a changelog just yet since this change is going into a feature branch. You can just ignore the automation bots message about it

Reply:
Ah ok got it